src/Tribe/Attendee_Registration/View.php

```diff
@@ -262,31 +67,14 @@ public function get_cart_url( $provider ) {
 	 * @return boolean|object The provider object or boolean false if none found.
 	 */
 	public function get_cart_provider( $provider ) {
-		if ( empty( $provider ) ) {
-			return false;
-		}
-
-		$provider_obj = false;
-
-		/**
-		 * Allow providers to include themselves if they are not in the above.
-		 *
-		 * @since 4.11.0
-		 *
-		 * @param string $provider A string indicating the desired provider.
-		 *
-		 * @return boolean|object The provider object or boolean false if none found above.
-		 */
-		$provider_obj = apply_filters( 'tribe_attendee_registration_cart_provider', $provider_obj, $provider );
+		try {
+			/* @var Tribe__Tickets_Plus__Attendee_Registration__View $view */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 7cde50dc50fba2e2a955468eb2f108a7</sub>